src/subspace_store.c

```diff
@@ -189,6 +190,7 @@ ts_subspace_store_add(SubspaceStore *store, const Hypercube *hc, void *object,
 	}
 
 	Assert(last != NULL && last->storage == NULL);
+	/* NOLINTNEXTLINE(clang-analyzer-core.NullDereference) */
```

Comment:
Not sure why you need this: there is an assert on the line before.

Reply:
Assertions should not be enabled in release builds. See my other comment on this.